pkg/cluster/k8sres.go

```diff
@@ -500,6 +503,13 @@ func (c *Cluster) generatePodTemplate(
 	return &template
 }
 
+func getWALBucketScopeSuffix(uid string) string {
+	if uid != "" {
+		return fmt.Sprintf("-%s", uid)
+	}
+	return ""
```

Comment:
Does it makes sense to return an empty string from this function? What if there would be an error, that would lead to the same empty suffix for different clusters?

Reply:
It could be perfectly fine to call this function to finding out a prefix to clone with an empty uid, allowing us to clone the cluster that has been started before we introduced the prefixes (i.e. cloning the cluster A from the cluster B using S3 bucket s3://... /spilo/B).